website/project/model.py

```diff
@@ -730,6 +731,23 @@ def category_display(self):
         """The human-readable representation of this node's category."""
         return self.CATEGORY_MAP[self.category]
 
+    # We need the following 2 properties in order to serialize related links in NodeRegistrationSerializer
+    @property
+    def registered_user_id(self):
```

Comment:
Yes, I really don't like having to do this, but DRF won't let us do `lookup_field_name='registered_user._id'; it only allows simple attribute access. If you have suggestions for an immediate fix, I am open to them.

FWIW, I do intend to address this limitation if we go with my proposed `RelationshipField`.